### PR TITLE
[misc] fix: drop verl patch

### PR DESCRIPTION
**Pull request by zhtmike**
### What does this PR do?
- Drop verl patch once it is ready

> Add **concise** overview of what this PR aims to achieve or accomplish. Reference related GitHub issues and PRs that help with the review.

### Checklist Before Starting

- [ ] Search for similar PRs. Paste at least one query link here: ...
- [ ] Format the PR title as `[{modules}] {type}: {description}` (This will be checked by the CI)
  - `{modules}` include `fsdp`, `vllm_omni`, `rollout`, `trainer`, `ci`, `training_utils`, `recipe`, `ray`, `worker`, `single_controller`, `misc`, `perf`, `model`, `algo`, `env`, `tool`, `ckpt`, `doc`, `data`, `cfg`, `reward`, `diffusion`, `omni`, `tests`, `docker`
  - If this PR involves multiple modules, separate them with `,` like `[diffusion, doc]`
  - `{type}` is in `feat`, `fix`, `refactor`, `chore`, `test`
  - If this PR breaks any API (CLI arguments, config, function signature, etc.), add `[BREAKING]` to the beginning of the title.
  - Example: `[BREAKING][diffusion, fsdp] feat: new rollout scheduler`

### Test

> For changes that can not be tested by CI (e.g., algorithm implementation, new model support), validate by experiment(s) and show results like training curve plots, evaluation results, etc.

### API and Usage Example

> Demonstrate how the API changes if any, and provide usage example(s) if possible.

```python
# Add code snippet or script demonstrating how to use this
```

### Design & Code Changes

> Demonstrate the high-level design if this PR is complex, and list the specific changes.

### Checklist Before Submitting

> [!IMPORTANT]
> Please check all the following items before requesting a review, otherwise the reviewer might deprioritize this PR for review.

- [ ] Read the [Contribute Guide](https://github.com/verl-project/verl-omni/blob/main/CONTRIBUTING.md).
- [ ] Apply [pre-commit checks](https://github.com/verl-project/verl-omni/blob/main/CONTRIBUTING.md#code-linting-and-formatting): `pre-commit install && pre-commit run --all-files --show-diff-on-failure --color=always`
- [ ] Add / Update the documentation.
- [ ] Add unit or end-to-end test(s) to [the CI workflow](https://github.com/verl-project/verl-omni/tree/main/.github/workflows) to cover all the code. If not feasible, explain why: ...

### CODE REVIEW

File: verl_omni/_patch.py
Comment (SamitHuang) [inline verl_omni/_patch.py:1]: Need to update the installation doc on verl commit id as well?

Comment (zhtmike) [reply verl_omni/_patch.py:1]: let me verify it first, I think there should be something missing; update in next PR

**Comment by zhtmike** [comment]
ready @SamitHuang 

```log
Test Results  —  2026-04-29 22:08:39
Repo: /scratch/fq9hpsac/mikecheung/gitlocal/verl-omni

ID    RESULT   ELAPSED   NAME
----  -------  --------  ----
0     PASS     74s       vllm-omni rollout
1     PASS     110s      diffusion agent loop
2     PASS     181s      visual reward manager
3     PASS     103s      diffusers FSDP engine
4     PASS     133s      FlowGRPO trainer e2e
------------------------------------------------------------------------------
  Total: 5  |  Passed: 5  |  Failed: 0  |  Skipped: 0
  Full logs: /scratch/fq9hpsac/mikecheung/gitlocal/verl-omni/logs/gpu_smoke/20260429_215837
------------------------------------------------------------------------------
```

File: verl_omni/utils/fsdp_utils.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                        f"{block_prefix}.{param_name}": (
+                            param.full_tensor().detach().cpu()
+                            if hasattr(param, "full_tensor")
+                            else param.detach().cpu()
+                        )
+                        for param_name, param in sub_lora_params.items()
+                    }
+                    lora_params.update(sub_lora_params)
+                    submodule._is_root = False
+                get_torch_device().empty_cache()
```
[the remark this answers is not in the document]
Comment (zhtmike) [reply verl_omni/utils/fsdp_utils.py:63]: This is borrowed from `verl`, need to do profile before dropping.

File: verl_omni/reward_loop/_patches.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+    def patched_compute_rm_score(self, data: DataProto) -> DataProto:
+        if self.reward_model_manager is not None:
+            self.reward_model_manager.wake_up()
+
+        chunks = data.chunk(len(self.reward_loop_workers))
+        outputs = ray.get(
+            [
+                worker.compute_score_batch.remote(chunk)
+                for worker, chunk in zip(self.reward_loop_workers, chunks, strict=True)
```
[the remark this answers is not in the document]
Comment (zhtmike) [reply verl_omni/reward_loop/_patches.py:103]: This is borrowed from `verl`, suggest not to change.
